core/src/main/java/com/airbnb/aerosolve/core/transforms/DeleteFloatFeatureFamilyTransform.java

```diff
@@ -1,19 +1,29 @@
 package com.airbnb.aerosolve.core.transforms;
 
 import com.airbnb.aerosolve.core.FeatureVector;
+
+import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
+
 import com.typesafe.config.Config;
 
 /**
- * "field1" specifies the float feature family to be deleted
+ * "field1" optionally specifies the float feature family to be deleted
+ * "fields" optionally specifies a list of float feature families to be deleted
  */
-
 public class DeleteFloatFeatureFamilyTransform implements Transform {
   private String fieldName1;
+  private List<String> fieldNames;
 
   @Override
   public void configure(Config config, String key) {
-    fieldName1 = config.getString(key + ".field1");
+    if (config.hasPath(key + ".field1")) {
+      fieldName1 = config.getString(key + ".field1");
+    }
+    if (config.hasPath(key + ".fields")) {
+      fieldNames = config.getStringList(key + ".fields");
+    }
```

Comment:
If there is no existing production job relying on `field1`, it seems we can get rid of it. When the user want to delete just one family, just put it in `fields` would be ok.